__mocks__/handlers/task-details.handler.ts

```diff
@@ -70,6 +70,37 @@ const taskDetailsHandler = [
             })
         );
     }),
+    rest.get(`${URL}/tasks/6KhcLU3yr45dzjQIVm0l/details`, (_, res, ctx) => {
```

Comment:
You can use wild cards and slugs, that way we can add different ids and different behavior in one handler itself.
Check the path params section in this doc https://mswjs.io/docs/basics/request-matching